src/ensembl/src/header/launchbar/LaunchbarButton.tsx

```diff
@@ -28,7 +28,7 @@ const LaunchbarButton: FunctionComponent<LaunchbarButtonProps> = (
     <ImageButton
       classNames={{
         [Status.DEFAULT]: styles.launchbarButtonImage,
-        [Status.ACTIVE]: styles.launchbarButtonSelectedImage,
+        [Status.UNSELECTED]: styles.launchbarButtonSelectedImage,
```

Comment:
That’s weird: why is UNSELECTED paired with the DEFAULT here, and not with SELECTED?